hacspec/examples/aes-gcm/aes.rs

```diff
@@ -238,7 +343,7 @@ fn test_kat_block1() {
         0x97
     ]);
 
-    let c = aes128_encrypt_block(key, msg);
+    let c = aes_encrypt_block(key, msg, 4, 10);
```

Comment:
This function doesn't exist?

Reply:
I ignored the test, but will correct them.